
---

int64 code
string message
temoto_core/Error[] error_stack
